custom_components/luxtronik/climate.py

```diff
@@ -53,14 +53,25 @@
 MIN_TEMPERATURE = 8
 MAX_TEMPERATURE = 28
 
-HVAC_ACTION_MAPPING: dict[str, str] = {
+HVAC_ACTION_MAPPING_HEAT: dict[str, str] = {
     LuxOperationMode.heating.value: HVACAction.HEATING.value,
-    LuxOperationMode.domestic_water.value: HVACAction.HEATING.value,
+    LuxOperationMode.domestic_water.value: HVACAction.IDLE.value,
```

Comment:
When hot water is produced, the underfloor heating also receives heat.
So the heating thermostat is active if domestic_water is active.